css/style.css

```diff
@@ -0,0 +1,1138 @@
+  align-items: center;
+  padding-right: 26px;
+}
+
+.wrapper__box-name {
+  display: flex;
+  font-style: normal;
+  font-weight: bold;
+  align-items: flex-end;
+  font-size: 17px;
+  line-height: 21px;
+  margin-top: 0;
+  color: white;
+}
+
+.box-name__subtitle {
+  font-size: 10px;
+  line-height: 12px;
+  opacity: .7;
+}
+
+.wrapper__box-stars {
+  display: flex;
+  margin-top: -20px;
+  margin-right: -4px;
+}
+
+.box-stars__img {
+  margin-right: 4px;
+}
+
+.aside__safe {
+  display: flex;
+  justify-content: space-between;
+}
+
+.aside__img-safe {
+  margin-top: 34px;
+  margin-left: 8px;
+}
+
+.safe__about {
+  color: white;
+  margin-top: 34px;
+  margin-bottom: 35px;
+  margin-right: 28px;
+}
+
+.aside__safe-stars {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 50px;
+  line-height: 61px;
+}
+
+.aside__safe-defense {
+  margin-top: -59px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 25px;
+  line-height: 30px;
+}
+
+.box-about__footer {
+  align-items: center;
+  margin-right: -13px;
+  margin-left: -20px;
+}
+
+.box-about__footer-one {
+  display: flex;
+}
+
+.footer__section {
+  display: inline-block;
+  z-index: 0;
+}
+
+.footer__section-title {
+  display: inline-block;
+  font-style: normal;
+  position: inherit;
+  color: #e51937;
+  font-weight: 600;
+  font-size: 200px;
+  opacity: .1;
+  margin: 100px 0 0 176px;
+  z-index: -100;
+}
+
+.section__img-car {
+  display: inline-block;
+  width: 105%;
+  margin-top: -173px;
+  margin-left: -100px;
+  z-index: 100;
+}
+
+.section__img-car-ipad {
+  display: none;
+}
+
+.footer__dev {
+  display: flex;
+  justify-content: flex-end;
+  margin: 77px 18px 0 0;
+}
+
+@media (max-width: 1270px) {
+  .container {
+    flex-direction: column;
+  }
+
+  .container__title {
+    margin: 70px 46px 6px 0;
+    text-align: right;
+  }
+
+  .container__content {
+    flex-direction: column;
+  }
+
+  .container__img {
+    margin: -100px 0 151px 0;
+  }
+
+  .container__img-phone {
+    display: none;
+  }
+
+  .container__aside {
+    margin-left: 0;
+    display: grid;
+    grid-template-rows: 1fr 108px;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .aside-info {
+    margin: -2px 34px 0 -16px;
+  }
+
+  .container__boxes {
+    margin: -12px 0 0 66px;
+  }
+
+  .container__box {
+    margin-bottom: 21px;
+  }
+
+  .box-about {
+    display: grid;
+    grid-column: 1/3;
+  }
+
+  .box-about__container {
+    justify-content: center;
+    margin: 0 64px 40px 0;
+  }
+
+  .container__aside-info {
+    margin-top: 24px;
+    margin-bottom: 58px;
+  }
+
+  .box-about__img-ellipse {
+    width: 31.18px;
+  }
+
+  .box-about__learn {
+    font-size: 16px;
+  }
+
+  .box-about__button {
+    width: 220.84px;
+    height: 55.21px;
+    margin: 0 0 12px 44px;
+  }
+
+  .footer {
+    flex-direction: column;
+  }
+
+  .footer__section {
+    order: 1;
+    margin-bottom: 28px;
+  }
+
+  .footer__aside {
+    order: 2;
+    max-width: 1300px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)